docs/Compile.rst

```diff
@@ -200,39 +280,141 @@ On Windows, DFHack replaces the SDL library distributed with DF.
 
 Dependencies
 ------------
-You will need some sort of Windows port of git, or a GUI. Some examples:
+You will need the following:
+
+* Microsoft Visual Studio 2010 SP1, with the C++ language
+* Git
+* CMake
+* Perl with XML::LibXML and XML::LibXSLT
+
+  * It is recommended to install StrawberryPerl, which includes both.
+
+Microsoft Visual Studio 2010 SP1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The free Express version is sufficient.
+
+You can grab it from `Microsoft's site <http://download.microsoft.com/download/1/E/5/1E5F1C0A-0D5B-426A-A603-1798B951DDAE/VS2010Express1.iso>`_.
+
+You'll also need the Visual Studio 2010 SP1 update, which is obtained from
+Windows Update. After installing Visual Studio, be sure to go to Windows Update
+and check for and install the SP1 update. If no update is found, check that
+your Windows Update settings include "Updates from all Microsoft products".
+
+You can confirm whether you have SP1 by opening the Visual Studio 2010 IDE
+and selecting About from the Help menu.  If you have SP1 it will have *SP1Rel*
+at the end of the version number, for example: *Version 10.0.40219.1 SP1Rel*
+
+It is vital that you use SP1 as, while building with the original release
+of Visual Studio 2010 (RTM) may succeed, it will result in non-working DFHack
+binaries that crash when connecting to Dwarf Fortress.
+
+Additional dependencies: installing with the Chocolatey Package Manager
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The remainder of dependencies - Git, CMake and StrawberryPerl - can be most
+easily installed using the Chocolatey Package Manger. Chocolatey is a
+\*nix-style package manager for Windows. Think "``apt-get`` for Windows."
+
+Chocolatey is the recommended way of installing the required dependencies
```

Comment:
Chocolatey itself is using 18.5MB of disk space on my system with 20 or so packages installed.

I'd guess it takes maybe five minutes to install including opening/closing the cmd.exe windows etc.  Definitely significantly less time than it takes to install the three individual dependencies, factoring in time to go to three URLs, find the download link, download them, double click on them, click Next through the various installer wizards, then manually add them to PATH where not done already (Perl definitely needs manually adding, CMake doesn't, not sure about Git.)   

I'd say it's maybe 5 minutes total time for the Chocolatey + all dependencies install, versus at least 10 for the manual method, maybe 15.

I'd say it has about the same impact in terms of space and time as Homebrew or MacPorts on OSX - really not any at all.

But I'll reword it to highlight it's not actually required. Maybe 'preferred' instead of recommended.